### DISCUSSION TITLE
Is there syntax highlighting for *.ju.ipynb files?

### DISCUSSION THREAD

**Discussion by xavierchen0** [kiyoon/jupynium.nvim#131] [Q&A] [answered]
Hi I recently installed Jupynium and really enjoy this workflow over other Jupyter neovim plugins. I was wondering if there is supposed to be syntax highlighting because right now, none of my code is highlighted.

Sample of code:
![image](https://github.com/user-attachments/assets/ac16a2e4-e0f2-4932-b1f3-58d3d4333626)

Configuration in ~/.config/nvim/lua/plugins/jupynium.lua:
```
return {
  {
    "kiyoon/jupynium.nvim",
    -- build = "pip3 install --user .",
    build = "conda run --no-capture-output -n jupynium pip install .",
    opts = function(_, opts)
      opts.python_host = { "conda", "run", "--no-capture-output", "-n", "jupynium", "python" }
    end,
  },
}
```

**Comment by kiyoon** [comment, marked as answer]
Hi, I think you're probably using the jupytext plugin to convert it to the jupytext format. But you'll have to change the filetype to python (`:set filetype=python`) and it will follow all of the python workflow correctly.

**Reply by xavierchen0** [reply]
I appreciate the fast response and for making this plugin!

**Reply by kiyoon** [reply]
Thank you! I hope it resolves your issue well :)

**Reply by kiyoon** [reply]
Another way is to convert ipynb to jupytext format with the CLI provided: `ipynb2jupytext file.ipynb`
